build.xml

```diff
@@ -262,8 +272,183 @@ compressStatic
       XML. -->
     <ant antfile="${toDir.base.path}${file.separator}article-map.xml" dir="${basedir}" 
          target="zipArticleXml" inheritRefs="true"/>
-    <echo>Done!</echo>
-    <echo>To zip up the entire static site directory, use the "compressStatic" build target.</echo>
+  </target>
+
+  <target name="generateSiteToGo" depends="generateSite"
+          description="Generate a static copy of DHQ (without search capability), and ZIP it for easy transfer to a server. This is NOT a standalone copy.">
+    <!-- Generate the ZIP file of the entire static site. -->
+    <antcall target="compressStatic"/>
+  </target>
+
+  <!--
+toDir=${toDir}
+toDir.base=dhq-static
+toDir.use=../dhq-static/dhq
+toDir.base.path
+toDir.static=dhq
+toDir.static.path=../dhq-static/dhq
+-->
+  
+  <target name="generateSearchable" depends="generateSite"
+          description="Generate a searchable static non-standalone copy of DHQ.">
+    <!-- Copy a few needed files over -->
+    <copy todir="${toDir.static.path}/data/uvepss">
+      <fileset dir="data/uvepss"/>
+    </copy>
```

Comment:
Do you mean in build-properties.xml?
In which case references would need to be `${toDir.base}/${toDir.search}`, right?

Reply:
@sydb Yes, sorry, I meant build-properties.xml. And build.xml would need to use `${toDir.static.path}/${toDir.search}` or `${toDir.use}/${toDir.search}`. I prefer the latter, but since the rest of the build target uses `${toDir.static.path}`, it'd be better to use that, and keep things consistent.